Installing on windows fails

I've installed this package in my project before but when trying to move to a new package manager and reinstall my packages , I get an error with the following

OS : Windows 11 22H2
Node version : 21.1.0
PNPM Version: 8.10.3

```
node_modules/node-unrar-js postinstall$ wget https://www.rarlab.com/rar/unrarsrc-6.1.7.tar.gz && tar -xf unrarsrc-6.1.7.tar.gz -C …
│ --2023-11-22 03:50:22--  https://www.rarlab.com/rar/unrarsrc-6.1.7.tar.gz
│ Resolving www.rarlab.com (www.rarlab.com)... 51.195.68.162
│ Connecting to www.rarlab.com (www.rarlab.com)|51.195.68.162|:443... connected.
│ HTTP request sent, awaiting response... 200
│ Length: 236798 (231K) [application/x-gzip]
│ Saving to: 'unrarsrc-6.1.7.tar.gz'
│      0K .......... .......... .......... .......... .......... 21%  161K 1s
│     50K .......... .......... .......... .......... .......... 43%  251K 1s
│    100K .......... .......... .......... .......... .......... 64%  227K 0s
│    150K .......... .......... .......... .......... .......... 86%  305K 0s
│    200K .......... .......... .......... .                    100%  433K=1.0s
│ 2023-11-22 03:50:25 (239 KB/s) - 'unrarsrc-6.1.7.tar.gz' saved [236798/236798]
│ tar: could not chdir to 'src/cpp'
```

This error hadn't happened before

cpu-features@0.0.2 install /app/prj/node_modules/cpu-features
> node-gyp rebuild

make: Entering directory '/app/prj/node_modules/cpu-features/build'
  ACTION Configuring dependencies /app/prj/node_modules/cpu-features/deps/cpu_features/build/Makefile
/bin/sh: 1: cmake: not found
make: *** [config_deps.target.mk:13: /app/prj/node_modules/cpu-features/deps/cpu_features/build/Makefile] Error 127
make: Leaving directory '/app/prj/node_modules/cpu-features/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (events.js:400:28)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:285:12)
gyp ERR! System Linux 5.10.102.1-microsoft-standard-WSL2
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /app/prj/node_modules/cpu-features
gyp ERR! node -v v14.21.3
gyp ERR! node-gyp -v v5.1.1
gyp ERR! not ok 

> node-unrar-js@2.0.1 postinstall /app/prj/node_modules/node-unrar-js
> wget https://www.rarlab.com/rar/unrarsrc-6.1.7.tar.gz && tar -xf unrarsrc-6.1.7.tar.gz -C src/cpp && rm -rf unrarsrc-6.1.7.tar.gz

--2023-11-24 08:52:57--  https://www.rarlab.com/rar/unrarsrc-6.1.7.tar.gz
Resolving www.rarlab.com (www.rarlab.com)... 51.195.68.162
Connecting to www.rarlab.com (www.rarlab.com)|51.195.68.162|:443... connected.
HTTP request sent, awaiting response... 200 
Length: 236798 (231K) [application/x-gzip]
Saving to: ‘unrarsrc-6.1.7.tar.gz’

unrarsrc-6.1.7.tar.gz               100%[==================================================================>] 231.25K  83.5KB/s    in 2.8s    

2023-11-24 08:53:03 (83.5 KB/s) - ‘unrarsrc-6.1.7.tar.gz’ saved [236798/236798]

tar: src/cpp: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now

npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! node-unrar-js@2.0.1 postinstall: `wget https://www.rarlab.com/rar/unrarsrc-6.1.7.tar.gz && tar -xf unrarsrc-6.1.7.tar.gz -C src/cpp && rm -rf unrarsrc-6.1.7.tar.gz`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the node-unrar-js@2.0.1 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2023-11-24T08_53_06_961Z-debug.log


I'm having the same issue, so I installed node-unrar-js@2.0.0 and it's installed

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -21,6 +21,7 @@ jobs:
           cache: "npm"
           node-version: 18.x
       - run: npm ci
+      - run: npm run downloadUnrarSrc
       - run: npm run build:release
       - run: npm test
       - run: npm run clean:test

--- a/README.md
+++ b/README.md
@@ -209,7 +209,7 @@ If you want to compile the module by yourself, please follow the steps below:
 
 - Download the c++ source of unRar library by:
 
-`npm run prepare`
+`npm run downloadUnrarSrc`
 
 - Build for debug:
 

--- a/package.json
+++ b/package.json
@@ -11,6 +11,7 @@
   "module": "./esm/index.esm.js",
   "scripts": {
     "prepare": "husky install",
+    "downloadUnrarSrc": "wget https://www.rarlab.com/rar/unrarsrc-6.1.7.tar.gz && tar -xf unrarsrc-6.1.7.tar.gz -C src/cpp && rm -rf unrarsrc-6.1.7.tar.gz && husky install",
     "clean": "rm -rf dist coverage dist_map esm",
     "clean:test": "rm -rf dist/test dist/build esm/test esm/build dist/**/*.map esm/**/*.map",
     "commit": "cz",
@@ -19,7 +20,6 @@
     "build:debug": "tsc && node ./dist/build/make.js",
     "prebuild:release": "npm run lint",
     "build:release": "npm run clean && tsc && node ./dist/build/make.js release && tsc -p tsconfig-esm.json && node ./dist/build/make.js release esm",
-    "postinstall": "wget https://www.rarlab.com/rar/unrarsrc-6.1.7.tar.gz && tar -xf unrarsrc-6.1.7.tar.gz -C src/cpp && rm -rf unrarsrc-6.1.7.tar.gz",
     "test": "mocha dist/test/*.spec.js",
     "test:debug": "npm run test -- --inspect-brk"
   },

